test/elixir/test/bulk_docs_test.exs

```diff
@@ -133,7 +133,8 @@ defmodule BulkDocsTest do
   @tag :with_db
   test "bulk docs raises error for transaction larger than 10MB", ctx do
     docs = [%{_id: "0", a: random_string(16_000_000)}]
-    old_size = Couch.get("/_node/node1@127.0.0.1/_config/couchdb/max_document_size").body
+#    old_size = Couch.get("/_node/node1@127.0.0.1/_config/couchdb/max_document_size").body
+    old_size = "8000000"
     set_config_raw("couchdb", "max_document_size", "67108864") # 64M
     resp = Couch.post("/#{ctx[:db_name]}/_bulk_docs", body: %{docs: docs})
     set_config_raw("couchdb", "max_document_size", old_size) # set back
```

Comment:
We can skip reading the old size and re-setting it as the default is 8Mb. We could even lower the random_string size a bit too.

Reply:
Thanks for the review